packages/emitter-framework/src/python/components/enum-declaration.tsx

```diff
@@ -0,0 +1,47 @@
+import { ComponentChild } from "#jsx/jsx-runtime";
+import { Enum } from "@typespec/compiler";
+import { Declaration } from "../framework/components/index.js";
+import { code } from "../framework/core/index.js";
+import { useNamePolicy } from "../framework/core/name-policy.js";
+import { BaseClasses } from "./base-classes.js";
+import { ConstantDeclaration } from "./constant-declaration.js";
+import { mapWithSep } from "./utils.js";
+
+/**
+ * Represents the properties for a class declaration.
+ */
+export interface EnumDeclarationModel {
+  /** The TypeSpec type this understands */
+  type: Enum;
+  /** Name of the class. If not provided, it will be inferred from the type name per naming policy. */
+  name?: string;
+  children?: ComponentChild[];
+}
+
+export function EnumDeclaration({ type, name, children }: EnumDeclarationModel) {
+  if (type.kind !== "Enum") return undefined;
+
+  const namer = useNamePolicy();
```

Comment:
do we need usePythonNamePolicy instead?

Reply:
It's the same policy but usePythonNamePolicy gives you better types.